mlrun/projects/project.py

```diff
@@ -747,23 +747,6 @@ def get_code_path(self):
         """Get the path to the code root/workdir"""
         return path.join(self.context, self.workdir or self.subpath or "")
 
-    def is_remote(self) -> bool:
-        """
-        Checks if the source of the project is remote.
-        A remote project can be considered as one of the following:
-            * GitHub project e.g., git://github.com/mlrun/project-demo.git
-            * Archive project e.g., "v3io:///projects/my-project/my-project.zip
-              (Archiving a project can be done by `project.export`)
-        """
-        source = self.source
-        if not source or source.startswith(".") or source.startswith("/"):
-            return False
-        return (
-            source.startswith("git://")
-            or source.endswith(".tar.gz")
-            or source.endswith(".zip")
-        )
```

Comment:
same here, what replaced these checks?

Reply:
` if "://" not in current_source`, this function was wrong implementation (added by me in the previous PR)